modules/react/select/stories/Select.stories.mdx

```diff
@@ -34,79 +34,209 @@ yarn add @workday/canvas-kit-react
+							{item}
+						</Select.Item>
+					);
+				}}
+			</Select.List>
+		</Select.Card>
+	</Select.Popper>
+</Select>
+Selected Value: {model.state.selectedIds[0]}
+</>
+```
 
-<ExampleCodeBlock code={Required} />
+#### Array of objects
+
+If you have an array of `objects`, the collection system needs an `id` to ensure items are
+registered and searchable. In most cases, the `id` should match your text value for each item. Any
+additional information like icons or if the item is disabled can exist on the object and be used in
+the render function.
+
+```tsx
+const model = useSelectModel({
+	items: [{id: 'Mobile' icon: mobileIcon},{id:'E-mail', icon: emailIcon}, {id:'Fax' icon: faxIcon}]
+})
+<>
+<Select model={model}>
+	<Select.Input id="dynamic-select" />
+	<Select.Popper>
+		<Select.Card maxHeight="200px">
+			<Select.List>
+				{item => {
+					return (
+						<Select.Item>
+							<Select.Item.Icon icon={item.icon} />
+							{item.id}
+						</Select.Item>
+					);
+				}}
+			</Select.List>
+		</Select.Card>
+	</Select.Popper>
+</Select>
+Selected Value: {model.state.selectedIds[0]}
+</>
+```
 
-### Error States
+<ExampleCodeBlock code={HoistedModel} />
 
-Set the `error` prop of the wrapping Form Field to `FormField.ErrorType.Alert` or
-`FormField.ErrorType.Error` to set the Select to the Alert or Error state, respectively. You will
-also need to set the `hintId` and `hintText` props on the Form Field to meet accessibility
-standards.
+### Error States
 
-The `error` prop may be applied directly to the Select with a value of `Select.ErrorType.Alert` or
-`Select.ErrorType.Error` if Form Field is not being used.
+Set the error prop of the wrapping `FormField` to `FormField.ErrorType.Alert` or
+`FormField.ErrorType.Error` to set the `<Select>` to the alert or error state, respectively. You
+will also need to set the `hintId` and `hintText` props on the `FormField` to meet accessibility
+standards. You must set an `id` attribute on the `Select.Input` element that matches the value of
+`inputId` set on the `FormField` element. These attributes ensure that the alert message is
+associated to the `Select` and read out by voiceover.
+
+**Note: The Select container component, `Select` must wrap the `FormField` element to ensure correct
+styling on the `Select.Input` element.**
+
+```tsx
+<Select items={options}>
+	<FormField label="Contact" inputId="contact-id-formfield">
+		<Select.Input id="contact-id-formfield" />
+		... select sub components
+	</FormfField>
+</Select>
+```
 
 #### Alert
 
+Use the alert state when a selection is valid but there is additional information.
+
 <ExampleCodeBlock code={Alert} />
 
 #### Error
 
+Use the error state when the selection is no longer valid.
+
+> **Note**
+>
+> The blade-old package has been moved to it's own repo.
+> [github.com/razorpay/blade-old](https://github.com/razorpay/blade-old) (Only accessible to Razorpay employees)
+>
+> It is under **maintenance** and it won't have any major releases.
```

Comment:
This is a strange note. An inadvertent copy-and-paste?

```suggestion
Use the error state when the selection is no longer valid.
```

Reply:
lol yes, I was testing out creating a note, sorry!